### PR TITLE
Validate login

### PR DESCRIPTION
**Pull request by sjlarson92**
- created new request to check user email and password
- receives a response and either redirects user on successful response or renders error message

### CODE REVIEW

File: src/__tests__/User/Login/LoginScreen.test.js [hunk 1]
```diff
@@ -60,74 +61,47 @@ describe('LoginScreen', () => {
...
-        .simulate('change', { target: { value: email } })
-      wrapper
-        .find({ 'data-testid': 'passwordInput' })
-        .simulate('change', { target: { value: password } })
-      wrapper.find({ 'data-testid': 'loginButton' }).simulate('click')
-      expect(dispatch).toHaveBeenCalledWith({ type: TYPES.LOGIN })
+    describe('when there is not an errorMessage', () => {
+      it('should not render errorMessage', () => {
+        useSelector.mockReturnValue('')
+        const newWrapper = shallow(<LoginScreen />)
```
Comment (lgc13) [inline src/__tests__/User/Login/LoginScreen.test.js:78]: when checking that an errorMessage does not exist, you should set it to null or undefined. Empty string is essentially the same as a full string

File: src/main/User/Login/authRequests.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import axios from 'axios'
+import * as TYPES from '../../storage/actions'
+
+export const validateLogin = (dispatch, history, email, password) => {
+  console.log('email: ', email)
```
Comment (lgc13) [inline src/main/User/Login/authRequests.js:5]: can we remove the console log?

File: src/main/User/Login/authRequests.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  const GATEWAY_URL = process.env.REACT_APP_GATEWAY_URL
+  axios
+    .post(`${GATEWAY_URL}/api/login`, null, {
+      auth: {
+        username: email,
+        password,
+      },
+    })
+    .then(response => {
+      console.log('response: ', response)
```
Comment (lgc13) [inline src/main/User/Login/authRequests.js:15]: console.log

File: src/main/User/Login/authRequests.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      dispatch({
+        type: TYPES.LOGIN,
+      })
+      history.push('/')
+    })
+    .catch(error => {
+      console.log('FAIL: Failed to login with error', error.message)
```
Comment (lgc13) [inline src/main/User/Login/authRequests.js:22]: console.log

File: src/__tests__/User/Login/LoginScreen.test.js (hunk 1 above)
Comment (sjlarson92) [reply src/__tests__/User/Login/LoginScreen.test.js:78]: fixed

File: src/main/User/Login/authRequests.js (hunk 2 above)
Comment (sjlarson92) [reply src/main/User/Login/authRequests.js:5]: done!

File: src/main/User/Login/authRequests.js (hunk 3 above)
Comment (sjlarson92) [reply src/main/User/Login/authRequests.js:15]: removed

File: src/main/User/Login/authRequests.js (hunk 4 above)
Comment (sjlarson92) [reply src/main/User/Login/authRequests.js:22]: also removed. where do these keep coming from? hahahahaha ....

**Pull request merged**